### PR TITLE
Introduced patternfly empty state for Baremetal inventory

### PR DESCRIPTION
**Pull request by belwell**
Implemented HostsEmptyState component for Baremetal inventory.
WIP due to need assistance with aligning the EmptyState component
to centre. Styling margin: 0 auto; works, so need to identify
a patternfly component or class that implements that styling.

### CODE REVIEW

File: src/components/HostsEmptyState.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
```
Comment (jtomasek) [inline src/components/HostsEmptyState.tsx]: Linter complains on unused imports here

File: src/components/HostsEmptyState.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  Button,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  Bullseye
+} from '@patternfly/react-core';
+import { CubesIcon } from '@patternfly/react-icons';
+
+class HostsEmptyState extends React.Component {
+  render() {
```
Comment (jtomasek) [inline src/components/HostsEmptyState.tsx]: Linter complains on missing return type
`render(): JSX.Element {`

**Comment by honza** [comment]
If you rebase your branch on top of master, you'll get a proper build failure here.

**Comment by knowncitizen** [comment]
> We are using https://prettier.io/ for code formatting. You can apply it either by running `yarn prettier` or add it to your editor for automatic formatting on save. Doing this will fix most of the linting errors.
> 
> You can use `<Bullseye>` component to align the HostsEmptyState to the center (https://patternfly-react.surge.sh/patternfly-4/layouts/bullseye) See #27. Don't worry about the scrollbar, I am fixing it in #27

Might be worth documenting the prettier stuff in the README, @jtomasek

**Comment by knowncitizen** [comment]
Nevermind, I did it ;) https://github.com/metalkube/facet/pull/33/files

File: src/components/HostsTable.tsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React, { FC, Fragment } from 'react';
 import { Table, TableHeader, TableBody } from '@patternfly/react-table';
 import { Bullseye } from '@patternfly/react-core';
 import { HostTableRows } from '../models/hosts';
+import { HostsEmptyState } from './HostsEmptyState';
```
Comment (jtomasek) [inline src/components/HostsTable.tsx]: HostsEmptyState is default export, it needs to be imported as:
`import HostsEmptyState from './HostsEmptyState';`

**Pull request merged**